CHANGES.rst

```diff
@@ -1105,6 +1105,9 @@ astropy.io.ascii
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Fix bug where manual fixes to invalid header cards were not preserved when
```

Comment:
This is still listed under 4.0.5. Do I have to move it as part of the manual backport?

Reply:
No, it should stay here just as you've done.